integration_tests/snaps/asset-tracking/snapcraft.yaml

```diff
@@ -9,3 +9,4 @@ parts:
   asset-tracking:
     plugin: nil
     stage-packages: ['hello=2.10-1']
+    build-packages: ['make=4.1-6']
```

Comment:
I'm going to run the zesty autopkgtests before this merges. Will this pass? :wink: .